ipa_packager.py

```diff
@@ -74,3 +74,28 @@ def package(self):
             ipa.add(lib_path, "SwiftSupport/%s" % os.path.basename(lib))
 
         ipa.package()
+
+def main(argv):
+    inputfile = ''
+    outputfile = ''
+    try:
+        opts, args = getopt.getopt(argv,"hi:o:",["ifile=","ofile="])
+    except getopt.GetoptError:
+        print 'python ipa_packager.py -i <AppBundle.app> -o <AppIPA.ipa>'
+        sys.exit(2)
+    for opt, arg in opts:
+        if opt == '-h':
+            print 'python ipa_packager.py -i <AppBundle.app> -o <AppIPA.ipa>'
+            sys.exit()
+        elif opt in ("-i", "--ifile"):
+            inputfile = arg
+        elif opt in ("-o", "--ofile"):
+            outputfile = arg
+         
```

Comment:
Would be great if it would fail and print out a message at this point if `inputfile` or `outputfile` are not set. instead of failing at a later step during its process.